recipes/pysbol/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "pysbol" %}
+{% set version = "2.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/SynBioDex/pySBOL/archive/{{ version }}.tar.gz
+  sha256: 153432071fdba11ad3612c183f589bee8d9855d088203d5be8832c8026c4e5ec
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install -vv .
+
+requirements:
+  host:
+    - python =3.7
```

Comment:
Looking at the `setup.py` it looks like this should probably be `python =2.7|>=3.6`

Reply:
Hi @synapticarbors ,

sorry for the long delays, we had to make some changes. This package is working only with python 3.7, the setup has been updated. I’m aware of the availability of pysbol2 but it doesn’t give us the expected results, that’s why we prefer for the moment using pysbol.

Thank you for your feedback and for your work.